### PR TITLE
Adding docs for the new Conda language/environment

### PR DESCRIPTION
**Pull request by danielfrg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ _asset_bundler_cache
 .jekyll-metadata
 .idea/
 CLIPBOARD.txt
+vendor/bundle
```
Comment (bnlb) [inline .gitignore]: If you want, we could make this even broader and just ignore the whole folder. The Jekyll docs [recommend](https://jekyllrb.com/tutorials/using-jekyll-with-bundler/) doing it that way, so I don't anticipate any issues.
```suggestion
vendor/
```

File: _pages/algorithm-development/languages/conda.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+author:
+image:
+    teaser: /language_logos/conda.svg
+---
+
+Conda is a popular package management system for managing dependencies of multiple languages,
+Python being the most popular one, and is specially used by data scientist to install
+dependencies for data science projects given the easy of installation of usually complicated
+libraries to install, specially those who depend on system-level packages and compilers.
+For more info about conda look at their [official documentation](https://docs.conda.io/en/latest/).
```
Comment (bnlb) [inline _pages/algorithm-development/languages/conda.md]: nit: all other cases of Conda are uppercased
```suggestion
For more info about Conda look at their [official documentation](https://docs.conda.io/en/latest/).
```

File: _pages/algorithm-development/languages/conda.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Algorithmia makes using conda environments easy by providing an Algorithmia environment
+with conda ready to be used and uses standard conda tooling such as the [`environment.yml` spec](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) to define the dependencies of your algorithm.
```
Comment (bnlb) [inline _pages/algorithm-development/languages/conda.md]:
```suggestion
with Conda ready to be used and uses standard conda tooling such as the [`environment.yml` spec](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) to define the dependencies of your algorithm.
```

File: _pages/algorithm-development/languages/conda.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+In order to use Conda to manage your dependencies you need to select it as a Language in the
+"Specify the environment section", you  can find "Conda" in the Dropdown.
+Then select the Python version you want to use.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/env_dropdown_conda.png" alt="Algorithm creation modal, Environment drop-down" class="screenshot">
+
+Algorithmia supports adding 3rd party dependencies coming including [main anaconda repository](https://repo.anaconda.com),
+[conda-forge](https://conda-forge.org), or any other valid conda repository.
```
Comment (bnlb) [inline _pages/algorithm-development/languages/conda.md]:
```suggestion
[conda-forge](https://conda-forge.org), or any other valid Conda repository.
```

**Review by bnlb — DISMISSED** [review]
Minor nits but no blockers.

**Comment by danielfrg** [comment]
Added you suggestions, thanks for the review!

**Review by bnlb — APPROVED** [review]
🐍

**Pull request merged**